mesh/pmesh.cpp

```diff
@@ -6305,6 +6308,7 @@ void ParMesh::ParPrint(ostream &os, const std::string &comments) const
    {
       os << "total_shared_faces " << sface_lface.Size() << '\n';
    }
+   os << "\n# group 0 has no shared entities\n";
```

Comment:
I noticed 2 old typos at L6294: `deliniate` -> `delineate`, `it's` -> `its`

Reply:
Fixed in 4943545f5.